### PR TITLE
pin AAT to PR 375

### PR DESCRIPTION
**Pull request by crimesp**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-4043

### Change description ###
Pin PR for performance testing in AAT enviro

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by mdayican — APPROVED** [review]

File: k8s/namespaces/em/em-hrs-api/aat.yaml [hunk 1]
```diff
@@ -2,8 +2,11 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: em-hrs-api
+  annotations:
+    fluxcd.io/tag.java: glob:pr-375-*
```
Comment (yogesh-hullatti) [inline k8s/namespaces/em/em-hrs-api/aat.yaml:6]: Needs few more lines here like this - https://github.com/hmcts/cnp-flux-config/blob/15f738efe5726d29dd8df3731be05c0eb353c2af/k8s/namespaces/em/em-anno/demo.yaml#L8

File: k8s/namespaces/em/em-hrs-api/aat.yaml [hunk 2]
```diff
@@ -2,8 +2,11 @@ apiVersion: helm.fluxcd.io/v1
...
 spec:
   values:
     java:
+      image: hmctspublic.azurecr.io/em/hrs-api:pr-375
```
Comment (yogesh-hullatti) [inline k8s/namespaces/em/em-hrs-api/aat.yaml:10]: Should this be pr-375-7e65cda-20210915170059  ?

File: k8s/namespaces/em/em-hrs-api/aat.yaml (hunk 1 above)
Comment (crimesp) [reply k8s/namespaces/em/em-hrs-api/aat.yaml:6]: it got rejected earlier with that line...

File: k8s/namespaces/em/em-hrs-api/aat.yaml (hunk 2 above)
Comment (crimesp) [reply k8s/namespaces/em/em-hrs-api/aat.yaml:10]: flux will update each image  i believe to the latest PR

File: k8s/namespaces/em/em-hrs-api/aat.yaml (hunk 1 above)
Comment (crimesp) [reply k8s/namespaces/em/em-hrs-api/aat.yaml:6]: https://github.com/hmcts/cnp-flux-config/runs/3621962647?check_suite_focus=true

docmosis
---
em-hrs-api'
+ HELMRELEASE_CHECK=em-hrs-api
+ '[' em-hrs-api == null ']'
+ '[' em-hrs-api == '' ']'
+ '[' 0 -eq 0 ']'
+ echo 'Non whitelisted HelmReleases found with hmcts.github.com/prod-automated annotation in k8s/aat/common-overlay'
+ exit 1
Error: Process completed with exit code 1.

**Comment by crimesp** [comment]
AAT can't have images pinned

**Pull request closed without merging**